run.py

```
@@ -169,97 +161,78 @@ def create_default_logo_config(config_path):
+    if reload:
+        cmd.append("--reload")
+
     try:
-        from app import app
-        app.run(
-            host=host,
-            port=port,
-            debug=debug,
-            use_reloader=debug
-        )
+        subprocess.run(cmd)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)